resources/lang/pt-br/contest.php

```diff
@@ -21,27 +21,27 @@
 return [
     'header' => [
         'small' => 'Compita de outras formas do que apenas clicar em círculos.',
-        'large' => 'Concursos Comunitários osu!',
+        'large' => 'Concursos comunitários osu!',
     ],
     'voting' => [
         'over' => 'A votação deste concurso terminou',
     ],
     'entry' => [
-        'login_required' => 'Por favor, Faça o login para participar deste concurso.',
+        'login_required' => 'Por favor, inicie a sessão para participar deste concurso.',
         'silenced_or_restricted' => 'Você não pode participar de concursos enquanto estiver restrito ou silenciado.',
         'preparation' => 'Estamos atualmente preparando este concurso. Aguarde pacientemente!',
-        'over' => 'Obrigado pela sua participação! As submissões foram encerradas para este concurso e a votação abrirá em breve.',
-        'limit_reached' => 'Você atingiu o limite de arquivos de submissão para este concurso',
-        'drop_here' => 'Deixe seu arquivo de submissão aqui',
+        'over' => 'Agradecemos a sua participação! As inscrições foram encerradas e a votação abrirá em breve.',
```

Comment:
"submissões" is even better than "inscrições", note: "inscrições" is not be the only way to contest...(possibility) keeps "submissões" because is more generic.
what was ended? "para este concurso" is needed.